test/test_property.py

```diff
@@ -151,6 +152,9 @@ def test_owned_object_singleton(self):
         cd.annotation = sbol.Identified('foo')
         self.assertEqual(type(cd.annotation), sbol.Identified)
 
+        # Test unsetting
+        cd.annotation = None
+
```

Comment:
We should probably add an assert to test that the value is actually set to `None` or some other null value.

Reply:
Done